src/lib/multipageloader.cc

```diff
@@ -104,6 +109,35 @@ QNetworkReply * MyNetworkAccessManager::createRequest(Operation op, const QNetwo
 		foreach (const HT & j, settings.customHeaders)
 			r3.setRawHeader(j.first.toLatin1(), j.second.toLatin1());
 	}
+
+    if(settings.clientSslKeyPath != NULL && settings.clientSslKeyPassword != NULL
+            && settings.clientSslCrtPath != NULL){
+        bool success = true;
+        QSslConfiguration sslConfig = QSslConfiguration::defaultConfiguration();
+
+        //key not supplied as string use path
+        QFile keyFile(settings.clientSslKeyPath);
+        success = keyFile.open(QFile::ReadOnly);
```

Comment:
Use `if (keyFile.open(QFile::ReadOnly) {` so that `success` is not required, and remove the comment (doesn't make sense) or change it.